src/wallet3/rpc/commands.h

```diff
@@ -102,7 +102,7 @@ namespace wallet::rpc {
     struct REQUEST
     {
       uint32_t account_index;             // Return balance for this account.
-      std::set<uint32_t> address_indices; // (Optional) Return balance detail for those subaddresses.
+      std::vector<uint32_t> address_indices; // (Optional) Return balance detail for those subaddresses.
```

Comment:
why to vector? set does not permit duplicates, is allowing duplicates desired?

Reply:
set wasn't playing nicely with the templates for parameter parsing. Didnt think preventing duplicates was critical here

Reply:
which template parameters specifically?

Reply:
https://github.com/darcys22/loki-core/blob/175a60a1be965ae5ec59d69546fedb547fe41cbc/src/rpc/common/param_parser.hpp#L112

We load into an "expandable_list" using `emplace_back` which set doesn't have. Could probably modify the template to create a new "insertable_list" aswell but it would be specifically for set I believe. 

But not having std::set available for RPCs parameters probably isn't going to be that big of an issue. Can de-duplicate elsewhere

Reply:
you could load it from a vector and insert the vector into the set which will deduplicate the collection.
the inverse is easy too.

Reply:
`our_set.insert(vec.begin(), vec.end());`

Reply:
Yeah easy, however right now this PR doesnt implement anything with the vector cause we dont have subaddresses yet. Left a TODO to come back to this